### PR TITLE
Nest a few fields together, retrieve keep activity

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim @camhashemi

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala [hunk 1]
```diff
@@ -114,6 +118,10 @@ class KeepInfoAssemblerImpl @Inject() (
       permissionCommander.getKeepsPermissions(keepSet, viewer)
     }
 
+    val activityFut = FutureHelpers.accumulateOneAtATime(keepSet) { keepId =>
+      keepCommander.getActivityForKeep(keepId, eventsBefore = None, maxEvents = config.numEventsPerKeep)
+    }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala:132]: @camhashemi is there a better way? I think we should work towards a batch endpoint.

File: kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala [hunk 2]
```diff
@@ -114,6 +118,10 @@ class KeepInfoAssemblerImpl @Inject() (
       permissionCommander.getKeepsPermissions(keepSet, viewer)
     }
 
+    val activityFut = FutureHelpers.accumulateOneAtATime(keepSet) { keepId =>
+      keepCommander.getActivityForKeep(keepId, eventsBefore = None, maxEvents = config.numEventsPerKeep)
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala]: not sure what the release timeline for these new endpoints is, but I've been hiding this behind the `activity_log` experiment whenever it's exposed to prod for performance / safety reasons.

File: kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala:132]: that's the goal, just working on getting the `n=1` case to-spec first.

File: kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala (hunk 2 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/assemblers/KeepInfoAssembler.scala]: Ah, fair enough, I can do the same here.

File: kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/keep/NewKeepInfo.scala [hunk 3]
```diff
@@ -20,7 +20,8 @@ final case class NewKeepInfo(
   source: Option[SourceAttribution],
   users: Seq[BasicUser],
   libraries: Seq[BasicLibrary],
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/shoebox/data/keep/NewKeepInfo.scala]: `recipients`?

**Pull request merged**